tests/test.js

```diff
@@ -201,6 +213,18 @@ describe('Autographa Test', () => {
       .getText('#chapterBtn').should.eventually.equal('1');
     });
 
+    it('should check verse existance after 3x, 4x click', () => {
+        const input = 'Check for verse5 visible after 3x 4x click';
+        return app.client.waitUntilWindowLoaded()
+        .waitForVisible("#versediv5", 20000)
+        .click("#versediv5")
+        .keys('Check for verse5 visible after 3x 4x click')
+        .pause(3000)
+        .waitForEnabled("#btn-3x", 20000)
+        .click("#btn-3x")
+        .getText("#v5").should.eventually.equal(input);
+    });
+
   	it('should change layout to 3x', () => {
```

Comment:
Remove this.

Reply:
And 'should change layout to 4x' test.

Reply:
I removed both '**should change layout to 4x**' and '**should change layout to 3x**' since we are doing same thing above

Reply:
I think we should not remove the above two test it affects the functionality  We should keep the above two test cases and add the new test case in the after **verse highlight check of 2x**